Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -24,44 +22,16 @@ public partial class TapToPlace : MonoBehaviour
         public string SavedAnchorFriendlyName = "SavedAnchorFriendlyName";
 
         /// <summary>
-        /// Keeps track of anchors stored on local device.
+        /// Manages persisted anchors.
         /// </summary>
-        WorldAnchorStore anchorStore = null;
-
-        /// <summary>
-        /// Locally saved wold anchor.
-        /// </summary>
-        WorldAnchor savedAnchor;
+        WorldAnchorManager anchorManager;
 
         bool placing = false;
 
         void Start()
         {
-            WorldAnchorStore.GetAsync(AnchorStoreReady);
-        }
-
-        /// <summary>
-        /// Called when the local anchor store is ready.
-        /// </summary>
-        /// <param name="store"></param>
-        void AnchorStoreReady(WorldAnchorStore store)
-        {
-            anchorStore = store;
-
-            // Try to load a previously saved world anchor.
-            savedAnchor = anchorStore.Load(SavedAnchorFriendlyName, gameObject);
-            if (savedAnchor == null)
-            {
-                // Either world anchor was not saved / does not exist or has a different name.
-                Debug.Log(gameObject.name + " : "+ "World anchor could not be loaded for this game object. Creating a new anchor.");
-
-                // Create anchor since one does not exist.
-                CreateAnchor();
-            }
-            else
-            {
-                Debug.Log(gameObject.name + " : " + "World anchor loaded from anchor store and updated for this game object.");
-            }
+            anchorManager = WorldAnchorManager.Instance;
+            anchorManager.AttachAnchor(this.gameObject, SavedAnchorFriendlyName);
```

Comment:
NIT: Worth adding a null check here and logging a Debug.LogError in case developer missed dragging the WorldAnchorManager somewhere.